nixos/community.tt

```diff
@@ -9,14 +9,14 @@
     well):</p>
```

Comment:
This diff is nothing but updating `<span class="muted">$</span> my command` to `<span class="command">my command</span>`